opentdf-dev.yaml

```diff
@@ -17,6 +17,16 @@ services:
       - kid: e1
         alg: ec:secp256r1
         legacy: true
+      - kid: e2
+        alg: ec:secp384r1
+      - kid: e2
+        alg: ec:secp384r1
+        legacy: true
```

Comment:
Do we need to legacy on everything?

Reply:
We probably should add a 'default' or 'current' boolean as well so we don't have to mark the same key twice in the keyring. At present, you have to list the same key twice to get it to show up in `PublicKey` and to be tested for kidless headers ('legacy' mode)

So we could replace:

```yaml
    keyring:
      - alg: ec:secp3841r1
        kid: a
      - alg: ec:secp3841r1
        kid: a
        legacy: true
```
with:

```yaml
    keyring:
      - alg: ec:secp3841r1
        kid: a
        current: true
        legacy: true
```

Reply:
We could also have legacy default to true, or current default to true, or maybe have current default to !legacy or something more confusing (or rename current to something else or have an enum for these options?)

Reply:
is there an action to be done in this PR?

Reply:
No we can update it later

Reply:
No we can update it later